src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -201,6 +201,7 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
 .step-icon {
   font-size: small;
   font-weight: bold;
+  pointer-events: none;
```

Comment:
I removed pointer events so these icons don't look/behave clickable.

Reply:
Also, doing this fixed their colour: they were previously blue (same as buttons) but are are now "gray7" as per template code.